docs/content/cli/upgrade.md

```diff
@@ -34,7 +34,7 @@ porter upgrade [INSTALLATION] [flags]
   porter upgrade --reference localhost:5000/ghcr.io/getporter/examples/kubernetes:v0.2.0 --insecure-registry --force
   porter upgrade MyAppInDev --file myapp/bundle.json
   porter upgrade --parameter-set azure --param test-mode=true --param header-color=blue
-  porter upgrade --credential-set azure --credential-set kubernetes
+  porter upgrade --cred azure --cred kubernetes
```

Comment:
I think the doc changes got reverted during the merge

Reply:
😭

Reply:
I pushed up a fix for the bad merge conflict resolution.